src/Files.Core/Data/Enums/LayoutGridViewIconHeightKind.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Core.Data.Enums
+{
+	public enum LayoutGridViewIconHeightKind
```

Comment:
```suggestion
	public enum GridViewSizeKind
```
This should have values from 0 - 12 but I'm not sure what to name them.